src/backbone.marionette.js

```diff
@@ -20,11 +20,11 @@ import View               from './view';
 import CollectionView     from './collection-view';
 import CompositeView      from './composite-view';
 import Behavior           from './behavior';
-import Behaviors          from './behaviors';
 import Region             from './region';
 import Application        from './application';
 import AppRouter          from './app-router';
 import MarionetteError    from './error';
+import behaviorsLookup    from './behaviors-lookup';
```

Comment:
any thoughts on putting this and features into like a config directory?  Not sure if there's really anything else that would qualify as config related things?